config.go

```diff
@@ -59,5 +61,11 @@ func Parse(path string) (*Config, error) {
 		return nil, fmt.Errorf("packages in %s must be alphabetically ordered", path)
 	}
 
+	if c.GodocServer == "" {
+		c.GodocServer = "https://godoc.org"
+	} else if strings.HasSuffix(c.GodocServer, "/") {
+		return nil, fmt.Errorf("godoc server address should not end up with a slash")
```

Comment:
Why not handle this for the user by just trimming the trailing "/" instead of returning error?

Reply:
+1. This can be an unconditional `strings.TrimSuffix(c.GodocServer, "/")`.

Reply:
It's better! Thanks.